ui/DesignSystem/Component/ModalLayout.svelte

```diff
@@ -3,6 +3,13 @@
   import { Icon } from "../Primitive";
 
   export let dataCy = null;
+
+  // close on ESC
```

Comment:
This comment should be closer to the `keyCode` making it clear that 27 is the escape key, otherwise a const more appropriatly named would also help communicating the intend.